src/main.rs

```diff
@@ -463,6 +472,7 @@ mod tests {
                 room: -1234567890i64
             }),
             run_interval: 999,
+            cron_interval: "59 23 * * *".to_owned(),
```

Comment:
Where practical, please separate out the `next_run_time` or similar method into a separate method so that you can write unit tests for it separately.